src/etl/molecular_interaction_etl.py

```diff
@@ -464,6 +464,11 @@ def get_generators(self, filepath, batch_size):  # noqa
 
                 # Skip commented rows.
                 if row[0].startswith('#'):
+                    reg = re.compile(r'# File generated \(UTC\): (.*)')
+                    match = reg.match(row[0])
+                    if match:
+                        date_produced = match.group(1)
+                        ETLHelper.load_release_info_from_args(logger=self.logger, provider='COMBINED', sub_type='INTERACTION-', date_produced=date_produced)
```

Comment:
Not sure on the subtype here... typically the FMS is looking for the format RELEASE_DATATYPE_SUBTYPE so with interaction the subtype includes the `-` and whatever is proceeded. Maybe I am not understanding this segment?

Reply:
cut and paste error, well I forgot to add the 'MOL' after the 'INTERACTION-'

Reply:
Oh ok... was wondering.